python/pyspark/sql/dataframe.py

```diff
@@ -1311,6 +1329,28 @@ def fill(self, value, subset=None):
     fill.__doc__ = DataFrame.fillna.__doc__
 
 
+class DataFrameStatFunctions(object):
+    """Functionality for statistic functions with :class:`DataFrame`.
+    """
+
+    def __init__(self, df):
+        self.df = df
+
+    def cov(self, col1, col2):
```

Comment:
Slightly weird, but probably better than the two falling out of sync.

You can also say 
```
:func:`DataFrame.cov` and :func:`DataFrameStatFunctions.cov` are aliases.
```